Interactive_3D_Vis.Rmd

```diff
@@ -0,0 +1,544 @@
+
```

Comment:
Please start the first line with "# your title". 
![1668732056569](https://user-images.githubusercontent.com/114460912/202590134-5282f4ef-9c79-4787-996f-8d6e9486727c.png)